adapters/repos/db/inverted/searcher_doc_bitmap.go

```diff
@@ -87,20 +90,21 @@ func (s *Searcher) docBitmapInvertedSet(ctx context.Context, b *lsmkv.Bucket,
 	limit int, pv *propValuePair,
 ) (docBitmap, error) {
 	out := newDocBitmap()
-	var readFn ReadFn = func(k []byte, ids [][]byte) (bool, error) {
-		for _, asBytes := range ids {
-			out.docIDs.Set(binary.LittleEndian.Uint64(asBytes))
+	var readFn ReadFn = func(k []byte, ids *sroar.Bitmap, op filters.Operator) (bool, error) {
+		out.docIDs.SetMany(ids.ToArray())
```

Comment:
since `ids` are now bitmap, 1st bitmap can be used to collect final results. same approach as in `RoaringSetReadFn`:

```
if isEmpty {
	out.docIDs = docIDs
	isEmpty = false
} else {
	out.docIDs.Or(docIDs)
}
```